### DISCUSSION TITLE
Gauge Step Implementation not found if steps are in different .js folder.

### DISCUSSION THREAD

**Discussion by prem1321** [getgauge/gauge#2073] [General]
The gauge specs automatically detects if step implementation files (.js) are within /tests folder. However it does not automatically detect test steps that are outside /tests folder.

**Comment by zabil** [comment]
The detection isn't automatic. This is a configured using `STEP_IMPL_DIR` [property](https://docs.gauge.org/configuration.html?os=macos&language=javascript&ide=vscode#language-plugin-configurations)
